kustomize/preprod/simple-app/secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-secret
+type: Opaque
+data:
+  MY_SECRET: echo -n "s3cr3t" | base64
```

Comment:
Does it work really ?

Reply:
Updated! Please re-review and send me feedbacks. Thanks!